Assets/Scripts/Collectables/HealthBasedDropper.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Actor;
+using UnityEngine;
+
+namespace Collectables
+{
+    /// <summary>
+    /// A dropper that drops drops based on the health of the entity it's bound to.
+    /// </summary>
+    public class HealthBasedDropper : Dropper
+    {
+        /// <summary>
+        /// The entity that this dropper is attached to. Used to keep track of health.
+        /// </summary>
+        public Living entity;
+        
+        /// <summary>
+        /// The health points at which to drop drops.
```

Comment:
I think it would be good to add:

`The health point thresholds at which to drop pickups.` (or drop drops 🤷 )

Reply:
This no longer applies with the refactor.